ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -139,6 +139,8 @@ QtObject {
         }
 
         switch(code) {
+        case Constants.routerErrorCodes.errGeneric:
+            return qsTr("generic error")
```

Comment:
are we going to display this to end user?

Reply:
@alaibe this is the case when something happens which we don't have a full control of, those are errors returned by the network for example, in that case we can say an error happened and that's all, but even though for the 2.30 I would go with providing details, even they are raw. So what user will see if opens details of the error is something like this ` test.api.status.im.error: hystrix: circuit open, test.api.status.im.error:  hystrix: circuit open, eth-archival.rpc.grove.city.error: execution reverted`.

Reply:
I'd really think we should be more open: like oops, unexpected error, pls retry in few minutes or reach out to us... kinda?

Reply:
@alaibe @alexjba I am pretty sure that many of those "generic errors" will end up in a display like this one:

<img width="589" alt="image" src="https://github.com/user-attachments/assets/f0601372-f73d-42b4-a548-526360bd6351">

but because of those rare cases we have to handle them on the client side generic error as well, so could you please suggest what exactly you would like to see in case of a generic error for:
- the title
- the description

at this moment the title is "generic error" and the details are in the form they are in the image above.

Reply:
Any suggestion? 
What should be displayed for the title and details in case of a generic error?
@alaibe @alexjba

Reply:
My suggestion for this would be this. It's quite the standard AFAIK:

- Title: "An unknown error has occurred. Please try again later."
- Description: as it is now.

Reply:
@alexjba updated, thanks.